player_darwin.c

```diff
@@ -0,0 +1,143 @@
+//
+// Created by Christopher Cooper on 7/12/17.
+//
+
+#include <sys/param.h>
+#include "player_darwin.h"
+
+#pragma mark - utility functions
```

Comment:
Hmm, as I think there are few use cases to edit `oto` with Xcode, let's remove this. Thanks.